Description: drop __proto__ calls in test
Author: Yadd <yadd@debian.org>
Forwarded: no
Last-Update: 2022-12-02

--- a/test/compilation.coffee
+++ b/test/compilation.coffee
@@ -105,8 +105,8 @@
 
 test "#1106: __proto__ compilation", ->
   object = eq
-  @["__proto__"] = true
-  ok __proto__
+  Object.setPrototypeOf @, {a: true}
+  ok Object.getPrototypeOf(@).a
 
 test "reference named hasOwnProperty", ->
   CoffeeScript.compile 'hasOwnProperty = 0; a = 1'
